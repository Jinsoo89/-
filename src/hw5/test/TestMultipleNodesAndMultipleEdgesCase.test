# Sample testing file.
# Tests that a graph can be created, multiple nodes and
# multiple edges. And nodes and edges can be printed out.

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add an edge
AddEdge graph1 n1 n2 edge1
AddEdge graph1 n2 n3 edge2
AddEdge graph1 n3 n1 edge3
AddEdge graph1 n3 n2 edge4
AddEdge graph1 n3 n3 edge5

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2
ListChildren graph1 n3
