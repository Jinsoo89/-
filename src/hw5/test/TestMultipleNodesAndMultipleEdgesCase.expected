# Sample testing file.
# Tests that a graph can be created, multiple nodes and
# multiple edges. And nodes and edges can be printed out.

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add an edge
added edge edge1 from n1 to n2 in graph1
added edge edge2 from n2 to n3 in graph1
added edge edge3 from n3 to n1 in graph1
added edge edge4 from n3 to n2 in graph1
added edge edge5 from n3 to n3 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1 n2 n3
the children of n1 in graph1 are: n2(edge1)
the children of n2 in graph1 are: n3(edge2)
the children of n3 in graph1 are: n1(edge3) n2(edge4) n3(edge5)
